website/docs/r/google_project_iam_role.html.markdown

```diff
@@ -0,0 +1,56 @@
+---
+layout: "google"
+page_title: "Google: google_project_iam_role"
+sidebar_current: "docs-google-project-iam-role"
+description: |-
+ Allows management of a customized Cloud IAM role.
+---
+
+# google\_project\_iam\_role
+
+Allows management of a customized Cloud IAM role. For more information see
+[the official documentation](https://cloud.google.com/iam/docs/understanding-custom-roles)
+and
+[API](https://cloud.google.com/iam/reference/rest/v1/organizations.roles).
+
+## Example Usage
+
+This snippet creates a customized IAM role.
+
+```hcl
+resource "google_project_iam_role" "myCustomRole" {
```

Comment:
nit: I don't think we _actually_ have an opinion on this, but I feel like I very rarely see camelcasing in resource names. Most of the examples I found were one word, or used hyphens.

Reply:
I am ok with renaming the resource name. However, the `role_id` cannot have an hyphen: `It doesn't match pattern [a-zA-Z0-9_\.]{3,64}.`